src/app/(main)/wines/page.tsx

```
@@ -35,59 +36,62 @@ const WinesPage = () => {
-                value={rating}
-                onChange={setRating}
-                readOnly={false}
-                size='md'
-              />
+    <main className='overflow-hidden pb-[4.5rem]'>
+      {/* 메인 콘텐츠 영역을 InnerContainer로 감쌉니다. */}
+      <InnerContainer>
+        <h1 className='sr-only'>와인 목록 페이지</h1>
+        <MonthlyWineSection />
+        <button onClick={() => open('test01')}>Open Modal 01</button>
+        <Modal modalId='test01' title='test' showCloseButton>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)